reference/configuration/framework.rst

```diff
@@ -4,28 +4,61 @@
 FrameworkBundle Configuration ("framework")
 ===========================================
 
-This reference document is a work in progress. It should be accurate, but
-all options are not yet fully covered.
-
 The FrameworkBundle contains most of the "base" framework functionality
-and can be configured under the ``framework`` key in your application configuration.
+and can be configured under the ``framework`` key in your application
+configuration. When using XML, you can use the
+``http://symfony.com/schema/dic/symfony`` namespace.
+
 This includes settings related to sessions, translation, forms, validation,
 routing and more.
 
+.. tip::
+
+   The XSD schema is available at
+   ``http://symfony.com/schema/dic/symfony/symfony-1.0.xsd``.
+
 Configuration
 -------------
 
 * `secret`_
 * `http_method_override`_
+* `trusted_proxies`_
 * `ide`_
 * `test`_
-* `trusted_proxies`_
-* `form`_
-    * enabled
-* `csrf_protection`_
-    * enabled
-    * field_name
+* `default_locale`_
+* `trusted_hosts`_
+* :ref:`form <reference-framework-form>`
+    * :ref:`enabled <reference-form-enabled>`
+    * `csrf_protection`_
```

Comment:
Unfortunately, @xabbuh is right :). These big updates are a pain to merge (but we should still do them).